tests/integration/utils/snapshots.js

```diff
@@ -13,7 +14,22 @@ const normalizers = [
   { pattern: /It should be one of.+/gm, value: 'It should be one of: *' },
 ]
 
-const normalize = (inputString) =>
-  normalizers.reduce((acc, { pattern, value }) => acc.replace(pattern, value), inputString)
+const optionalNormalizers = {
+  // File paths
+  filePath: { pattern: /(^|[ "'(=])((?:\.{0,2}|([A-Z]:)|file:\/\/)(\/[^ "')\n]+))/gm, value: '/file/path' },
```

Comment:
NIT: would love to see a link to a regex 101 with some samples for this pattern.

Not everybody is capable of reading regexes so well

Reply:
This was taken from https://github.com/netlify/build/blob/main/packages/build/tests/helpers/normalize.js#L46.

I personally don't like the approach of relying on regular expressions to normalise the output of CLI/Netlify Build in snapshots, but that's our reality right now, and I don't have the bandwidth to add regex101 links to every expression.